pom.xml

```diff
@@ -742,6 +742,8 @@
             </property>
           </systemProperties>
           <runOrder>alphabetical</runOrder>
+          <!-- to avoid losing the focus on MacOS when a new forked jvm is created -->
+          <argLine>-Djava.awt.headless=true</argLine>
```

Comment:
maven-hpi-plugin as well?

(I've had this system property defined in all my run configurations for ages as it's super annoying on MacOS)